conans/client/cache/remote_registry.py

```diff
@@ -110,6 +113,10 @@ def update(self, remote_name, url=None, secure=None, disabled=None, index=None,
             remotes.insert(index, remote)
             self._remotes = {r.name: r for r in remotes}
 
+        if filters is not None:
+            remote.filters = remote.filters or []
+            remote.filters.extend(filters)
```

Comment:
The update shouldn't be an ``extend`` but a replacement. Otherwise it is not possible to remove things